docs/make.jl

```diff
@@ -18,6 +21,14 @@ quantum_pages = map(readdir(joinpath(example_dir, "quantum1d"))) do dir
     return joinpath("examples", "quantum1d", dir, "index.md")
 end
 
+# bibliography
+bibpath = joinpath(@__DIR__, "src", "assets", "mpskit.bib")
+bib = CitationBibliography(bibpath; style=:authoryear)
+
+# interlinks
+links = InterLinks("TensorKit" => "https://jutho.github.io/TensorKit.jl/stable/",
+                   "TensorOperations" => "https://jutho.github.io/TensorOperations.jl/stable/")
```

Comment:
```suggestion
links = InterLinks("TensorKit" => "https://jutho.github.io/TensorKit.jl/stable/",
                   "TensorOperations" => "https://jutho.github.io/TensorOperations.jl/stable/",
                   "KrylovKit" => "https://jutho.github.io/KrylovKit.jl/stable/")
```

KrylovKit links would also be nice (see below).